src/app-layout/__integ__/app-layout-drawers.test.ts

```diff
@@ -128,49 +144,71 @@ for (const visualRefresh of [true, false]) {
-
-        await page.dragAndDrop(wrapper.findActiveDrawerResizeHandle().toSelector(), -100);
-        await expect(page.getSplitPanelWidth()).resolves.toEqual(newSplitPanelWidth);
-        await expect(page.getActiveDrawerWidth()).resolves.toBeGreaterThan(originalDrawerWidth);
-      })
+      setupTest(
+        { visualRefresh, splitPanelPosition: 'side', screenSize: { ...viewports.desktop, width: 1800 } },
+        async page => {
+          await page.openFirstDrawer();
+          await page.openSplitPanel();
+
+          const originalSplitPanelWidth = await page.getSplitPanelWidth();
+          const originalDrawerWidth = await page.getActiveDrawerWidth();
+          await page.dragAndDrop(wrapper.findSplitPanel().findSlider().toSelector(), 100);
+
+          const newSplitPanelWidth = await page.getSplitPanelWidth();
+          expect(newSplitPanelWidth).toBeLessThan(originalSplitPanelWidth);
+          await expect(page.getActiveDrawerWidth()).resolves.toEqual(originalDrawerWidth);
+
+          await page.dragAndDrop(wrapper.findActiveDrawerResizeHandle().toSelector(), -100);
+          await expect(page.getSplitPanelWidth()).resolves.toEqual(newSplitPanelWidth);
+          await expect(page.getActiveDrawerWidth()).resolves.toBeGreaterThan(originalDrawerWidth);
+        }
+      )
     );
 
     test(
       'updates side split panel position when using different width drawers',
-      setupTest({ splitPanelPosition: 'side', screenSize: { ...viewports.desktop, width: 1500 } }, async page => {
-        await page.openFirstDrawer();
-        await page.openSplitPanel();
-        await expect(page.isExisting(wrapper.findSplitPanel().findOpenPanelSide().toSelector())).resolves.toEqual(true);
-
-        await page.openThirdDrawer();
-        await expect(page.isExisting(wrapper.findSplitPanel().findOpenPanelBottom().toSelector())).resolves.toEqual(
-          true
-        );
-
-        await page.openFirstDrawer();
-        await expect(page.isExisting(wrapper.findSplitPanel().findOpenPanelSide().toSelector())).resolves.toEqual(true);
-      })
+      setupTest(
+        { visualRefresh, splitPanelPosition: 'side', screenSize: { ...viewports.desktop, width: 1500 } },
+        async page => {
+          await page.openFirstDrawer();
+          await page.openSplitPanel();
+          await expect(page.isExisting(wrapper.findSplitPanel().findOpenPanelSide().toSelector())).resolves.toEqual(
+            true
+          );
+
+          await page.openThirdDrawer();
+          await expect(page.isExisting(wrapper.findSplitPanel().findOpenPanelBottom().toSelector())).resolves.toEqual(
+            true
+          );
+
+          await page.openFirstDrawer();
+          await expect(page.isExisting(wrapper.findSplitPanel().findOpenPanelSide().toSelector())).resolves.toEqual(
+            true
+          );
+        }
+      )
     );
   });
 }
+
+describe('Visual refresh only', () => {
```

Comment:
This is the only test I've added - the changes to the rest of the tests are just adding a {visualRefresh} to the setup, since I moved it out of the loop.